benchmark.py

```diff
@@ -105,32 +111,41 @@ def Profile(scenario_config, output_dir, dry_run=False):
 
   subprocess_to_pid = {}
```

Comment:
Sounds like subprocess_to_pid is rather pid_to_subprocess

Reply:
Good point!

Reply:
It does! Done.